mapbox_gl_web/lib/src/mapbox_map_controller.dart

```diff
@@ -80,6 +81,27 @@ class MapboxMapController extends MapboxGlPlatform
     await link.onLoad.first;
   }
 
+  void _onStyleImageMissing(dynamic event) {
+    _map.addImage(event.id, {
+      "data": Uint8List.fromList([0, 0, 0, 0]),
+      "height": 1,
+      "width": 1
+    });
+    if (this.onStyleImageMissing != null) {
+      this.onStyleImageMissing.call(event.id);
+      return;
+    }
+    var density = context['window'].devicePixelRatio ?? 1;
+    var imagePath = density == 1
+        ? '/assets/assets/symbols/custom-icon.png'
+        : '/assets/assets/symbols/$density.0x/custom-icon.png';
```

Comment:
enforcing the use of a statically defined custom image png seems like a bad idea